src/Components/Server/src/Circuits/CircuitJSComponentInterop.cs

```diff
@@ -9,29 +9,20 @@
 
 namespace Microsoft.AspNetCore.Components.Server.Circuits
 {
-    /// <summary>
-    /// Intended for framework use only. Not supported for use from application code.
-    /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
-    public class CircuitJSComponentInterop : JSComponentInterop
+    internal class CircuitJSComponentInterop : JSComponentInterop
```

Comment:
This no longer needs to be public because it's no longer called *directly* through JS interop. It's now called via the new `WebRendererInteropMethods` thing.